resolwe_bio/descriptors/super_sample.yml

```diff
@@ -0,0 +1,263 @@
+        - label: Adrenal gland
+          value: adrenal_gland
+        - label: Testis
+          value: testis
+        - label: Vein
+          value: vein
+        - label: Placenta
+          value: placenta
+        - label: Thyroid gland
+          value: thyroid_gland
+        - label: Ovary
+          value: ovary
+        - label: Thymus
+          value: thymus
+        - label: Breast
+          value: breast
+        - label: Artery
+          value: artery
+        - label: Lymphoid tissue
+          value: lymphoid_tissue
+        - label: Nerve
+          value: nerve
+        - label: Bone marrow
+          value: bone_marrow
+        - label: Eye
+          value: eye
+        - label: Uretar
+          value: uretar
+        - label: Vagina
+          value: vagina
+        - label: Spinal cord
+          value: spinal_cord
+        - label: Mouth
+          value: mouth
+        - label: Urinary bladder
+          value: urinary_bladder
+        - label: Bronchus
+          value: bronchus
+        - label: Lymphatic vessel
+          value: lymphatic_vessel
+        - label: Skeleton
+          value: skeleton
+        - label: Tongue
+          value: tongue
+        - label: Pericardium
+          value: pericardium
+        - label: Trachea
+          value: trachea
+        - label: Nose
+          value: nose
+        - label: Other
+          value: other
+        - label: N/A
+          value: na
+    - name: primary_cell_line
+      label: Primary cell line
+      type: basic:string
+    - name: cell_line
+      label: Cell line
+      type: basic:string
+    - name: strain
+      label: Strain
+      type: basic:string
+    - name: genotype
+      label: Genotype
+      type: basic:string
+    - name: biosample_treatment
+      label: Biosample treatment
+      type: basic:string
+    - name: treatment_protocol
+      label: Treatment protocol
+      type: basic:string
+    - name: platform
+      label: Platform
+      type: basic:string
+      choices:
+        - label: NextSeq 500
+          value: nextseq_500
+        - label: HiSeq 2500
+          value: hiseq_2500
+        - label: HiSeq 2000
+          value: hiseq_2000
+        - label: NovaSeq 6000
+          value: novaseq_6000
+    - name: molecule
+      label: Library made from / Molecule
+      type: basic:string
+      choices:
+        - label: Total RNA
+          value: total_rna
+        - label: PolyA RNA
+          value: polya_rna
+        - label: Cytoplasmic RNA
+          value: cytoplasmic_rna
+        - label: Nuclear RNA
+          value: nuclear_rna
+        - label: Genomic DNA
+          value: genomic_dna
+        - label: Protein
+          value: protein
+        - label: Other
+          value: other
+    - name: library_depleted_in
+      label: Library depleted in
+      type: basic:string
+    - name: library_treatment
+      label: Library treatment
+      type: basic:string
+    - name: replication_type
```

Comment:
What is this field for? In general, I think we lack `description` fields in this schema.